hwdb/70-mouse.hwdb

```diff
@@ -342,8 +342,6 @@ mouse:usb:v046dp402d:name:Logitech M560:
 mouse:usb:v046dpc52b:name:Logitech Unifying Device. Wireless PID:402d:
  MOUSE_DPI=1000@125
 
-# Logitech V220 Cordless Optical Mouse
-mouse:usb:v046dpc51b:name:Logitech USB Receiver:
```

Comment:
c51b seems to be a unifying receiver PID. For the c52b ID we have the kernel extract the actual device and present it as such, @bentiss do you know what the deal is where for the c51b? I can't find it in the kernel sources, is this a different model?